packages/vx-marker/src/markers/Line.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Marker, { MarkerProps } from './Marker';
+
+export interface MarkerLineProps {
+  id: string;
+  size?: number;
+  stroke?: string;
```

Comment:
do we need to defined `stroke` and `fill` since they are part of `MarkerProps`? 

could still keep the logic for pulling them out of `restProps` to ensure that `fill` is defined and `stroke="none"` in the component.